omnibus-arbitrum-mainnet.toml

```diff
@@ -1,21 +1,25 @@
 name = "synthetix-omnibus"
-version = "7"
+version = "18"
 description = "Includes the full synthetix system with configurations applied"
 preset = "main"
 include = [
     "tomls/core.toml",
     "tomls/markets/spot-factory.toml",
+    "tomls/omnibus-arbitrum-mainnet/perps-highcap-factory.toml",
     "tomls/omnibus-arbitrum-mainnet/permit-deniers.toml",
 
     "tomls/omnibus-arbitrum-mainnet/permissions.toml",
     "tomls/omnibus-arbitrum-mainnet/pools/spartan-council.toml",
 
     # USDC
     "tomls/oracles/pyth-usdc.toml",
-    "tomls/omnibus-arbitrum-mainnet/spot/usdc.toml",
     "tomls/omnibus-arbitrum-mainnet/collaterals/usdc.toml",
     "tomls/omnibus-arbitrum-mainnet/rewards/ARB_rewards_for_USDC_lp.toml",
 
+    # DAI
```

Comment:
Dai is not needed on mainnet.

Reply:
We just cannot remove it from testnet and stuck with it :(

Reply:
removed